docker/gcp_pudl_etl.sh

```diff
@@ -109,6 +116,9 @@ if [[ $ETL_SUCCESS == 0 ]]; then
     ETL_SUCCESS=${PIPESTATUS[0]}
```

Comment:
If we replace `success` file with, say, json file that has some runtime statistics from how the data was generated, would we want to include it in distributed files, or also not?

Reply:
I think a little build metadata file would be worth including. We're distributing the `pudl-etl.log` file. It would kind of be a structured accessory to that output.